fluffy.py

```diff
@@ -40,7 +40,7 @@ def article(topic=None, company=None):
     ))
     company = company or "ACME"
 
-    yield from intoduction(topic)
+    yield from introduction(topic, company)
```

Comment:
I get an `UnboundLocalError` if the program is called as: `$ python3 fluffy.py name`.